lib/generators/draft/model/model_generator.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
-
-require "rails/generators/active_record/model/model_generator"
+require 'rails/generators/named_base'
 
 module Draft
-  class ModelGenerator < ActiveRecord::Generators::ModelGenerator
-    source_root ActiveRecord::Generators::ModelGenerator.source_root
+  class ModelGenerator < Rails::Generators::NamedBase
+    argument :attributes, type: :array, default: [], banner: "field[:type][:index] field[:type][:index]"
+    
+    def generate_model
+      invoke 'model'
```

Comment:
Should we include [thoughtbot's rubocop](https://github.com/thoughtbot/guides/blob/master/style/ruby/.rubocop.yml) on this project to catch style?